LongWarOfTheChosen/Src/LW_OfficerPack_Integrated/Classes/X2StrategyElement_LW_OTS_OfficerStaffSlot.uc

```diff
@@ -108,7 +108,7 @@ static function X2DataTemplate CreateOTS_OfficerTrainingSecondSlotUpgradeTemplat
 
 static function OTS_LWOfficerTrainingUpgradeAdded(XComGameState NewGameState, XComGameState_FacilityUpgrade Upgrade, XComGameState_FacilityXCom Facility)
 {
-	Facility.PowerOutput += Upgrade.GetMyTemplate().iPower;
+	//Facility.PowerOutput += Upgrade.GetMyTemplate().iPower;
```

Comment:
I don't like commenting out lines without some explanatory text unless I'm doing it temporarily. Is there any reason to keep this line?